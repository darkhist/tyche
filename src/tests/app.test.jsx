import React from 'react';
import { shallow } from 'enzyme';

import App from '../components/app';

describe('App', () => {
  it('should render without crashing', () => {
    const app = shallow(<App />);
    expect(app).toMatchSnapshot();
  });
});
