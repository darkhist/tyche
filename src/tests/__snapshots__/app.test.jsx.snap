// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render without crashing 1`] = `
<Fragment>
  <Memo(GlobalStyleComponent) />
  <header>
    <Title />
    <Info />
  </header>
  <main>
    <Generator />
  </main>
</Fragment>
`;
